### ISSUE TITLE
Slow shutdown of ES on Windows 8.1 with roms on networkdrives

### ISSUE THREAD

**Issue by troelsdk** [Aloshi/EmulationStation#398]
I have appr. 6 emulators with a few thousand roms installed in ES2. The emulators are located locally and the roms on a SAMBA share on a NAS. 

When I quit ES2 it takes appr. 10 minutes before it is shutdown completely and I can see my network is working sending and receiving around 2mb/s until it's done.

Is this normal behaviour and what is it doing on the network drives when I quit?

I hope somebody has an idea, as this is rendering ES2 almost useless.

**Comment by Aloshi** [comment]
Is your `~/.emulationstation` folder on the local drive? Usually when exiting the bottleneck is writing the `gamelist.xml` files, which is currently super inefficient. Those files are usually in `~/.emulationstation/gamelists/[system]/gamelist.xml`. The `gamelistdb` branch aims to fix this by using a database instead, so you could try that instead if you're feeling risky (to import your current gamelist.xmls, you can use "Scraper -> Import XML").

**Comment by troelsdk** [comment]
Yes the gamelist.xml's are on a local SSD disk. Is it reading all the rom-folders when I quit?

I would love to try the gamelistdb version, but I gave up on compiling it on windows, as I couldnt get all the dependent packages to work while using cmake.

**Comment by troelsdk** [comment]
Aloshi - if you have a compiled win version lying around I would love to give it a try to see if it helps :-)

/Troels
